### PR TITLE
yazi: use builtin cd

### PR DESCRIPTION
**Pull request by khaneliman**
cd conflicts with aliases that dont support -- syntax

### Description

Enabling the zoxide --cmd cd will break this alias. Need to use the builtin cd behavior. 

### Checklist

- [x] Change is backwards compatible.

- [x] Code formatted with `./format`.

- [x] Code tested through `nix-shell --pure tests -A run.all` or `nix develop --ignore-environment .#all` using Flakes.

- [x] Test cases updated/added. See [example](https://github.com/nix-community/home-manager/commit/f3fbb50b68df20da47f9b0def5607857fcc0d021#diff-b61a6d542f9036550ba9c401c80f00ef).

- [ ] Commit messages are formatted like

    ```
    {component}: {description}

    {long description}
    ```

    See [CONTRIBUTING](https://nix-community.github.io/home-manager/#sec-commit-style) for more information and [recent commit messages](https://github.com/nix-community/home-manager/commits/master) for examples.

- If this PR adds a new module

  - [ ] Added myself as module maintainer. See [example](https://github.com/nix-community/home-manager/blob/068ff76a10e95820f886ac46957edcff4e44621d/modules/programs/lesspipe.nix#L6).

### CODE REVIEW

**Comment by rycee** [comment]
Thanks for the contribution. I'll let @XYenon, as maintainer, have a look.

**Review by XYenon — APPROVED** [review]
LGTM

**Comment by rycee** [comment]
Thanks! Merged to master now 🙂

**Pull request merged**
